knative-operator/Gopkg.toml

```diff
@@ -39,14 +39,10 @@
   name = "knative.dev/operator"
   version = "=v0.17.2"
 
-[[override]]
+[[constraint]]
```

Comment:
What's the difference between `override` and `constraint` and why has it changed here ?

Reply:
override is technically to deeply override something through other deps, but in this case we're actually constraining what **we** need.